e2e/fixtures/python/relative/subpackage2/import_from_other.py

```diff
@@ -0,0 +1,12 @@
+# @OctoLinkerResolve(<root>/python/relative/subpackage1/classes.py)
+from ..subpackage1.classes import MyClass1
+
+# @OctoLinkerResolve(<root>/python/relative/subpackage1/classes.py)
+from ...relative.subpackage1 import classes
+#    ^ should resolve folder        ^ should resolve file
```

Comment:
i dont know what to do here, i commented on this one case but there are similar cases everywhere

Reply:
Without making any changes to the current test code you could try using both `@OctoLinkerResolve()` and `@OctoLinkerResolveAbove()` so the first comment would check for the first link on the line below it, and the second comment will check for the second link on the line above it.

An example of this is:

```python
# @OctoLinkerResolve(<root>/python/relative/subpackage1)
from ...relative.subpackage1 import classes
# @OctoLinkerResolveAbove(<root>/python/relative/subpackage1/classes.py)
```

If there's more than two links per line then this wouldn't work. If we do need to expand the test api then maybe something like this could work?

```python
# @OctoLinkerResolveList(foo.py;bar.py;baz.py)
from ...relative.subpackage1 import classes
```

Reply:
Thanks for this. Ideally OctoLinker (and the tests) should resolve each segment of the entire subpackage path (and allow for that to be shown in the tests) but for now this works.